lib/manager/components/FeedSourceTableRow.js

```diff
@@ -338,13 +391,15 @@ class FeedActionsDropdown extends PureComponent<Props> {
     }
   }
 
-  render () {
-    const {
-      feedSource,
-      project,
-      user
-    } = this.props
-
+  /**
+   * This method is extracted variable generation which had to be split
```

Comment:
There was another instance of this sentence that doesn't make sense that I think you fixed.